flightgear.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-FG_ROOT=/app/share/flightgear FG_AIRCRAFT=~/.fgfs/Aircraft/ FG_SCENERY=~/.fgfs/Scenery fgfs --launcher "$@"
+FG_ROOT=~/.fgfs FG_AIRCRAFT=$FG_ROOT/Aircraft/ FG_SCENERY=$FG_ROOT/Scenery fgfs --launcher "$@"
```

Comment:
```suggestion
FG_ROOT=~/.fgfs fgfs --launcher "$@"
```

Those are the defaults anyway so you can remove them.